h/views/api/__init__.py

```diff
@@ -1,6 +1,11 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+API_VERSIONS = ["v1"]
+API_VERSION_DEFAULT = "v1"
```

Comment:
I was actually thinking maybe this should be called LATEST_STABLE_API_VERSION / something similar? Yes it is the default but more importantly it's intended to change overtime as we continue to release new versions of the api so I feel like having latest stable in there is more self documenting for future devs.